tests/testConstantSettings.php

```diff
@@ -13,7 +13,7 @@
  */
 class TestConstantSettings extends WP_Auth0_Test_Case {
 
-	use domDocumentHelpers;
+	use DomDocumentHelpers;
```

Comment:
Did an external dependency change the casing on this?

Reply:
No, was having trouble with the auto-loader picking up the file.